mmv1/products/iam3/FoldersPolicyBinding.yaml

```diff
@@ -47,6 +47,7 @@ async:
   error:
     path: 'error'
     message: 'message'
+  include_project: true
```

Comment:
This doesn't break anything but doesn't entirely do the right thing- it should be using the quota project, `billingProject`, but instead uses a blank `project` value (i.e. no quota project).

What did this fix for you? I'm having trouble understanding why it was needed.

Reply:
i worked with @slevenick quite a bit on this one. basically there are 3 resources, folder_policy_binding, organizations_policy_binding, projects_policy_binding that all bind to a folder/org/project to a principal_access_policy resource. to use the project (versus creating a new one), this was necessary to ensure the tests pass for folders and orgs.

Reply:
Got it!